evc-sumo/docker/install.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+#  Copyright (C) 2023 LEIDOS.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+set -e
+
+#This token is required to run the script
+evc_token=""
+
+while [[ $# -gt 0 ]]; do
```

Comment:
If you define the environment variable like the comment above, this arg-parsing while block should be unecessary. The defined environment variable based on the docker ARG will just propagate into this file and be available as a regular environment variable.

I'd also recommend deleting the above `evc_token` definition in line 20 so that you're not overwriting the value of that environment variable.

Reply:
Thank you for the suggestion Kyle, I followed the comment above and removed these codes